logger/logger.go

```diff
@@ -50,26 +51,27 @@ func RequestFromContext(ctx context.Context) (req *Request) {
 }
 
 // LogUltronEx logs a msg to UltronEx local file
-func LogUltronEx(msg *UltronExMsg) {
+func LogUltronEx(msg UltronExMsg) {
 	logger := loggers[logTypeUltronex]
-	if logger != nil && msg != nil {
+	if logger != nil && msg != (UltronExMsg{}) {
+		time.Sleep(slackPostingMsgLimitTime)
```

Comment:
just wondering, there is another application that calls the slack API right? so there is still a small chance of dropped message if different applications sends a lot of requests simultaneously to `ultronex` at the same time?

Reply:
Yes, I just realize it too.
We need to change the place where message got sent to slack.